pkg/injector/sidecar/pod_patch_test.go

```diff
@@ -14,9 +14,13 @@ limitations under the License.
 package sidecar
 
 import (
+	"encoding/json"
 	"strconv"
 	"testing"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	jsonpatch "github.com/evanphx/json-patch"
```

Comment:
```suggestion
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	jsonpatch "github.com/evanphx/json-patch"
```

Gotta make the linter happy...